### PR TITLE
dir: add page

### PR DESCRIPTION
**Pull request by amarjeetsinha**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by marchersimon** [comment]
What is going on with the branches here? Where is the `pages/dir` branch coming from and why does it say that you committed to it? I'm confused

**Comment by marchersimon** [comment]
Never mind, just found out what's going on (<https://github.com/tldr-pages/tldr/pull/6809#issuecomment-934712487>)

File: pages/linux/dir.md [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
 > More information: <https://manned.org/dir>.
 
-- Displays all the hidden files along with two files denoted by "." and "..":
+- List all files, including hidden files:
 
-`dir -a`
+`dir -all`
 
-- Displays author of all the files. -l is required to display the contents in the form of a list:
+- List files including the author (`-l` is required):
```
Comment (marchersimon) [inline pages/linux/dir.md:10]:
```suggestion
- List files including their author (`-l` is required):
```

File: pages/linux/dir.md [hunk 2]
```diff
@@ -3,22 +3,22 @@
...
 
 `dir -l --author`
 
-- Ignore files describe by shell PATTERN while list the contents of a directory:
+- List files excluding those that match a specified pattern:
```
Comment (marchersimon) [inline pages/linux/dir.md:14]: What kind of pattern? A blob pattern, regex, etc...

**Review by marchersimon — COMMENTED** [review]
Just two small suggestions

**Comment by marchersimon** [comment]
@navarroaxel do you plan to merge this PR into `pages/dir` and then merge `pages/dir` into `main`?

File: pages/linux/dir.md (hunk 2 above)
Comment (amarjeetsinha) [reply pages/linux/dir.md:14]: Yes Blob

**Comment by amarjeetsinha** [comment]
@navarroaxel @marchersimon Have a look on the edit, I hope this suites.

**Pull request merged**